src/views/admin/KlasseDetailsView.vue

```diff
@@ -117,12 +121,26 @@
   const onSubmit: (e?: Event | undefined) => Promise<Promise<void> | undefined> = handleSubmit(async () => {
     if (selectedSchule.value && selectedKlassenname.value) {
       if (organisationStore.currentOrganisation) {
-        await organisationStore.updateOrganisation(currentOrganisationId, selectedKlassenname.value);
+        try {
+          await organisationStore.updateOrganisationById(currentOrganisationId, selectedKlassenname.value);
+        } catch {
+          creationErrorText.value = t(`admin.klasse.errors.${organisationStore.errorCode}`);
+          creationErrorTitle.value = t(`admin.klasse.title.${organisationStore.errorCode}`);
+        }
       }
       resetForm();
     }
   });
 
+  async function deleteKlasse(organisationId: string): Promise<void> {
```

Comment:
Sollte das hier auch "deleteKlasseById" heißen oder ist das in dem Fall egal?

Reply:
Ja, danke!

Reply:
Schadet nicht!